lib/node_modules/@stdlib/string/base/replace-after/test/test.js

```diff
@@ -0,0 +1,150 @@
+	t.end();
+});
+
+tape( 'the function replaces the substring after the first occurrence of a specified search string', function test( t ) {
+	var replacement;
+	var expected;
+	var actual;
+	var str;
+
+	str = 'beep boop';
+	replacement = 'foo';
+	actual = replaceAfter( str, replacement, ' ' );
+	expected = 'beep foo';
+	t.strictEqual( actual, expected, 'returns expected value' );
+
+	str = 'beep boop';
+	replacement = 'foo';
+	actual = replaceAfter( str, replacement, 'p' );
+	expected = 'beepfoo';
+	t.strictEqual( actual, expected, 'returns expected value' );
+
+	str = 'Hello, World!';
+	replacement = 'foo';
+	actual = replaceAfter( str, replacement, 'o' );
+	expected = 'Hellofoo';
+	t.strictEqual( actual, expected, 'returns expected value' );
+
+	t.end();
+});
+
+tape( 'the function replaces the substring after the first occurrence of a specified search string (Unicode characters)', function test( t ) {
+	var replacement;
+	var expected;
+	var actual;
+	var str;
+
+	str = 'beep 😀 boop 😀 baz';
+	replacement = 'foo';
+	actual = replaceAfter( str, replacement, '😀' );
+	expected = 'beep 😀foo';
+	t.strictEqual( actual, expected, 'returns expected value' );
+
+	str = '🤖 Robot army 🤖!';
+	actual = replaceAfter( str, replacement, '🤖' );
+	expected = '🤖foo';
+
+	str = '🐺 Wolf brothers 🐺';
+	replacement = 'foo';
+	actual = replaceAfter( str, replacement, 'o' );
+	expected = '🐺 Wofoo';
+
+	t.end();
+});
+
+tape( 'the function replaces the substring after the first occurrence of a specified search string (custom start index)', function test( t ) {
+	var replacement;
+	var expected;
+	var actual;
+	var str;
+
+	str = 'beep boop baz';
+	replacement = 'foo';
+	actual = replaceAfter( str, replacement, ' ', 6 );
+	expected = 'beep boop foo';
+	t.strictEqual( actual, expected, 'returns expected value' );
+
+	str = 'beep boop baz';
+	replacement = 'foo';
+	actual = replaceAfter( str, replacement, 'p', 6 );
+	expected = 'beep boopfoo';
+
+	str = 'beep boop baz';
+	replacement = 'foo';
+	actual = replaceAfter( str, replacement, 'beep', -2 );
+	expected = 'beepfoo';
+
+	str = 'beep boop baz';
+	replacement = 'foo';
+	actual = replaceAfter( str, replacement, 'beep', 20 );
+	expected = 'beep boop baz';
+
+	t.end();
+});
+
+tape( 'the function returns the input string unchanged if the search string is not found', function test( t ) {
+	var replacement;
+	var expected;
+	var actual;
+	var str;
+
+	str = 'beep boop';
+	replacement = 'foo';
+	actual = replaceAfter( str, replacement, 'z' );
+	expected = 'beep boop';
+	t.strictEqual( actual, expected, 'returns expected value' );
+
+	str = 'beep boop';
+	replacement = 'foo';
+	actual = replaceAfter( str, replacement, 'baz' );
+	expected = 'beep boop';
+	t.strictEqual( actual, expected, 'returns expected value' );
+
+	t.end();
+});
+
+tape( 'the function returns the replacement string if the search string is the empty string', function test( t ) {
```

Comment:
I wouldn't expect this. Similar to `replace-before`, I'd expect the input string to be returned. I believe we discussed something similar when adding `replace-before`.

Reply:
Since an empty string would be found at index 0, I thought we can replace the string with replacement after the index 0. In case of replace before, there was nothing before index 0 to be replaced, hence so.

Reply:
Corrected it since I was already asked to return input string unchanged if search is empty